spark8t/lib/process_settings.py

```diff
@@ -0,0 +1,17 @@
+from argparse import Namespace
+
+from spark8t.cli import defaults
+from spark8t.domain import Defaults
+from spark8t.services import AbstractKubeInterface, KubeInterface, LightKube
+
+
+def get_kube_interface(args: Namespace) -> AbstractKubeInterface:
```

Comment:
**todo** we should probably provide also the defaults as argument here. We should really not import this from cli, as this creates a circular dependency (cli depends on lib which depends on cli)

Reply:
That's a valid point, my bad, I overlooked something on this level. Got it right after following up on your comment.

This may encourage one of the proposed strucutures I've listed in response to the abovementioned thread.